packages/flutter/lib/src/widgets/default_text_editing_shortcuts.dart

```diff
@@ -258,6 +259,30 @@ class DefaultTextEditingShortcuts extends StatelessWidget {
   // The macOS shortcuts uses different word/line modifiers than most other
   // platforms.
   static final Map<ShortcutActivator, Intent> _macShortcuts = <ShortcutActivator, Intent>{
+    const SingleActivator(LogicalKeyboardKey.keyX, meta: true): const CopySelectionTextIntent.cut(SelectionChangedCause.keyboard),
+    const SingleActivator(LogicalKeyboardKey.keyC, meta: true): CopySelectionTextIntent.copy,
+    const SingleActivator(LogicalKeyboardKey.keyV, meta: true): const PasteTextIntent(SelectionChangedCause.keyboard),
+    const SingleActivator(LogicalKeyboardKey.keyA, meta: true): const SelectAllTextIntent(SelectionChangedCause.keyboard),
+    const SingleActivator(LogicalKeyboardKey.keyZ, meta: true): const UndoTextIntent(SelectionChangedCause.keyboard),
+    const SingleActivator(LogicalKeyboardKey.keyZ, shift: true, meta: true): const RedoTextIntent(SelectionChangedCause.keyboard),
+
+    // On desktop these keys should go to the IME when a field is focused, not to other
```

Comment:
On all desktop platforms or only on Mac?

Reply:
Only on Mac (the comment is inside _macShortcuts), but not on web on mac (hence "desktop").